e2e/utils.go

```diff
@@ -572,6 +573,112 @@ func validatePVCClone(sourcePvcPath, clonePvcPath, clonePvcAppPath string, f *fr
 	validateRBDImageCount(f, 0)
 }
 
+func validateController(f *framework.Framework, pvcPath, appPath, scPath string) error {
```

Comment:
this can probably be placed in a new file, `e2e/rbd_controller.go`. Assuming more tests will be added at a later time, for bugfixes and/or features.

Reply:
kept it here as we will add support for cephfs later.